docs/modules/operation/pages/events/sources/send-event.adoc

```diff
@@ -0,0 +1,95 @@
+
+[[send-event]]
+= send-event.pl utility
+
+The send-event.pl utility lets you create and send an event to {page-component-title} without using the OpenNMS component.
+It is most frequently used to reload daemon configuration without having to restart {page-component-title}.
+You can also use the send-event.pl utility as an automation tool to trigger events as part of continuous integration scripts.
+
+For example:
+
+* create a new suspect event every time you add a new resource
+* clear out alarms
+* open a ticket in your integrated ticketing system
+* order new ink cartridges when your printer is out of toner
+
+Format: `bin/send-event.pl <uei> [host:port] [options]`
+
+The `host:port` defines the hostname and port to send the event; by default the local host.
+
+Find UEIs and their associated parameters in the `$\{OPENNMS_HOME}/etc/eventd-configuration.xml` file.
+See also xref:events/event-configuration.adoc#ga-events-event-configuration[Event Configuration].
+
+.`send-event.pl` parameters
+[options="header, autowidth"]
+[cols="3,3"]
+|===
+
+| Name/Alias
+| Description
+
+2+|*Required*
+
+| uei
+| The unique event identifier (UEI) indicates the type of event.
+View UEIs in the eventd-configuration.xml file.
+You can also define your own UEIs.
+
+2+|*Optional*
+
+| -d (--description)
+| A description for the event browser.
+
+| -f (--ifindex)
+| ifIndex of the associated service.
+
+| -h (--help)
+| Help message for the script.
+(The content from this table.)
+
+| -i (--interface)
+| IP address of associated interface.
+
+| -l (--logmsg)
+| A short log message for the event browser (secure field by default).
+
+| -n (--nodeid)
+| Database ID of associated node (or use foreignSource, foreignId parameters).
+
+| -p (-parameter)
+| Parameter in key=value form.
+For example, `-p 'url http://www.opennms.com/'`
+
+| -s (--service)
+| Name of the associated service
+
+| -x (--severity)
+| Severity of the event: +
+1=Indeterminate +
+2=Cleared +
+3=Normal +
+4=Warning +
+5=Minor +
+6=Major +
+7=Critical +
+
+| -t (--timezone)
+| The time zone you are in.
+
+| -v (--verbose)
+| Print raw XML that is generated.
+
+| -V (--version)
+| Print version and exit successfully.
+
+|===
+
+Note that parameter descriptions use the percentage sign (%) as a replacement variable.
```

Comment:
Same here. The percentage sign itself is not a replacement variable. It is used as a kind of delimiter to mark a replaceable variable.